ci-operator/step-registry/openshift/configure-cincinnati/openshift-configure-cincinnati-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Setting the cluster to Cincinnati instance: $CINCINNATI_URL"
+oc patch clusterversion version --patch '{"spec":{"upstream":"'"$CINCINNATI_URL"'"}}' --type=merge
+
+if [[ ! "$CHANGE_CHANNEL_BASE" =~ (stable|eus|fast|candidate) ]]; then
+    echo "CHANGE_CHANNEL_BASE is '$CHANGE_CHANNEL_BASE' which is not one of stable, eus, fast, or candidate. Skipping channel update."
+    exit
+fi
+
+current_channel="$(oc get clusterversion version -o jsonpath='{.spec.channel}')"
+if [[ "$current_channel" =~ (stable|eus|fast|candidate)-4\.([0-9]+) ]]; then
+    echo "Cluster is subscribed to channel: $current_channel"
+    desired_channel="${current_channel//${BASH_REMATCH[1]}/$CHANGE_CHANNEL_BASE}"
+    echo "Setting the cluster to use channel: $desired_channel instead of $current_channel"
+    oc adm upgrade channel "$desired_channel"
+else
+    echo "Cluster is currently not subscribed to a know version channel. Skipping channel update."
```

Comment:
nit: probably worth logging `$current_channel` in here, in case folks think "ah, I should extend the step to cover that channel set too"